features/space/impl/src/main/kotlin/io/element/android/features/space/impl/SpaceFlowNode.kt

```diff
@@ -120,7 +120,7 @@ class SpaceFlowNode(
                         backstack.pop()
                     }
 
-                    override fun navigateToSpaceInfo() {
+                    override fun navigateToEditDetails() {
```

Comment:
Unrelated to this PR? Let me know if you still want to merge it.

Reply:
Oooooops, good catch!